### PR TITLE
Reserve balance alerts

### LINKED ISSUE #22: Migrate Reserve Balance Alerts to Aegis
We have a #alerts-reserve channel on Discord that receives notifications when a given reserve assets it running low on balance potentially risking liveness of all pools trading against this asset:

Nobody seems to know where these alerts are coming from, it is possible that they're still running on Nadiem's private Google Cloud.

Additionally, not all reserve assets seem to be covered by these mysterious alerts. Our reserve ran out of USDT today and we didn't get any alerts before users started complaining about it.

**We should migrate these alerts to Aegis so they're under our control and can be managed as code.**

### Acceptance Criteria
- [ ] Every reserve asset has a configured grafana alert for when it's low on balance
- [ ] The grafana alerts should run into a separate Discord channel (either re-use #alerts-reserve but with a new webhook, or create a new channel)
- [ ] The grafana alerts have clear action items listed in the alert message, i.e. "a) Find XXX liquidity via reserve.mento.org b) Initiate Reserve MultiSig Transaction to swap and/or bridge liquidity to the [Reserve contract]"(https://celoscan.io/address/<reserve-addrses>)"
- [ ] The grafana alerts are deployed via Terraform

### PR DESCRIPTION
**Pull request by chapati23**
### Description
Fixes #22 

Migrated Nadiem's hand-rolled Google Script alerts into Aegis.

### Design Problem during implementation
I ran into an issue during implementation. So far, the Prometheus metric name was just `$funcName`, which worked fine until now.

However, I needed to add more `balanceOf` metrics, but this time on other contracts. And if we send four metrics, all called `balanceOf` to Prometheus, it turns out only one gets created because it likely sees the others as duplicates.

### Solution
I dabbled with a few different approaches, but the most straightforward one I found was just to include the contract name in the Prometheus metric ~~`$funcName`~~ -> **`$contractName_$funcName`** (underscore because dots are forbidden in Prometheus metric names).

This made the most sense to me because 
a) it eliminates the issue of not being able to have multiple metrics sharing the same function name
b) it makes it very clear in grafana which metric we're dealing with (i.e. in the query explorer you can select `USDC_balanceOf` and it's immediately clear what balance we're interested in here)

### How to test
- [x] Check out this branch locally
- [x] Run `pnpm run dev` to check it's still running and there's no errors in the terminal
- [x] Run `pnpm test` to see tests are green
- [x] Check out the new Grafana alerts, i.e. https://clabsmento.grafana.net/alerting/grafana/adzkr06itle68c/view?returnTo=%2Falerting%2Flist
- [x] Check the last messages in #🏦-reserve-alerts on discord to see how the alerts will look
- [x] Review the code

### CODE REVIEW

File: config.yaml [hunk 1]
```diff
@@ -1,5 +1,7 @@
 global:
   vars:
+    Reserve: '0x9380fA34Fd9e4Fd14c06305fd7B6199089eD4eb9'
```
Comment (nvtaveras) [inline config.yaml:3]: not related to this PR but do you think we should just keep a single yaml config file? feels weird to be basically duplicating it all the time 😅

File: terraform/grafana-folders.tf [hunk 2]
```diff
@@ -1,7 +1,11 @@
...
+  title = "Aegis"
+}
+
 resource "grafana_folder" "oracle_relayers" {
   title = "Oracle Relayers"
 }
 
-resource "grafana_folder" "aegis" {
-  title = "Aegis"
+data "grafana_folder" "reserve" {
```
Comment (nvtaveras) [inline terraform/grafana-folders.tf:9]: what's the difference here between `resource` and `data` for the folders?

**Review by nvtaveras — APPROVED** [review]
great work!

File: terraform/grafana-folders.tf (hunk 2 above)
Comment (chapati23) [reply terraform/grafana-folders.tf:9]: there was a pre-existing `reserve` folder in grafana already that was manually created sometime, `data` just means "use this resource that's already there" (whereas `resource` means "create this new resource")

File: config.yaml (hunk 1 above)
Comment (chapati23) [reply config.yaml:3]: fair point, tbh i never questioned this myself, can't think of a good reason to have separate files.

@bayological anything speaking against removing `config.local.yml`?

Comment (bayological) [reply config.yaml:3]: I agree with @nvtaveras here. I see no reason to keep it around.

**Review by bayological — APPROVED** [review]
Nice work 👍

File: config.yaml (hunk 1 above)
Comment (chapati23) [reply config.yaml:3]: aight, it's gone!

**Pull request merged**